lib/llm/src/kv_router/metrics_aggregator.rs

```diff
@@ -71,7 +76,14 @@ pub async fn collect_endpoints(
         .filter(|e| e.subject.starts_with(subject))
         .collect::<Vec<_>>();
     if endpoints.is_empty() {
-        tracing::debug!("Metrics endpoint not visible yet");
+        // Only print it once, we poll while the worker starts
```

Comment:
Just curious - if you were to stop/remove a worker, and then later start it again (same component name), would these still not print?

Probably not important, but just thinking it may be a bit unintuitive if silent in that scenario.

When working on the metrics binary, I would usually start it, it would be polling on some endpoint, and then I would iterate on the worker side while developing, often starting/stopping it. So if I wanted those debug logs after the first startup, may be unexpected if it it was silent.

Was the debug log too noisy?

Reply:
It logs every second I think, so yeah quite noisy. I see your point though.